Blink_ActiveWorkGoesHere/Animation.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Animation class. Defines the type of input neccessary to add a character animation.
+ */
+/**Animation class handles passing back animations for incoming spritesheets. */
+class Animation
+{
+    /**
+    * Single constructor takes in various information about the incoming spritesheet, as
+    * well as allowing the caller to define if it loops, how fast the default framerate is,
+    * and the scale of the output animation compaired to what is provided.
+    * 
+     * @param {any} spriteSheet
+     * @param {any} frameWidth
+     * @param {any} frameHeight
+     * @param {any} sheetWidth
+     * @param {any} frameDuration
+     * @param {any} frames
+     * @param {any} loop
+     * @param {any} scale
+     */
```

Comment:
Forgot to fill in these parameter fields for the jdoc. I'll do that on next check-in.